ui/apps/platform/src/Containers/Integrations/IntegrationTilesPage/IntegrationTilesPage.js

```diff
@@ -58,22 +60,15 @@ const IntegrationTilesPage = ({
         }
     }
 
-    function getIsIntegrationFeatureFlagEnabled(integration) {
-        if (integration.featureFlagDependency) {
-            return isBackendFeatureFlagEnabled(featureFlags, integration.featureFlagDependency);
-        }
-        return true;
-    }
-
     function renderIntegrationTiles(source) {
         return (
             integrationsList[source]
                 // filter out non-visible integrations
                 .filter((integration) => {
-                    const isIntegrationFeatureFlagEnabled =
-                        getIsIntegrationFeatureFlagEnabled(integration);
-                    if (!isIntegrationFeatureFlagEnabled) {
-                        return false;
+                    if (typeof integration.featureFlagDependency === 'string') {
```

Comment:
Do we consider it possible for `integration.featureFlagDependency` to be empty string `''`? 

If I understand JS correctly(which is a big chance I don't), in this case `if (integration.featureFlagDependency)` would return false, when `typeof integration.featureFlagDependency === 'string'` would return true.